### PR TITLE
feat: Update transfer dialogs design

### PR DESCRIPTION
**Pull request by gonzamontiel**
- [x] General dialog changes / refactor
- [x] To Pendulum Completed Dialog (both themes)
- [x] To Pendulum Cancelled Dialog (both themes)
- [x] To Pendulum Pending Dialog (both themes)

### CODE REVIEW

**Comment by gonzamontiel** [comment]
Somehow, after merging staging the styles in the branch are broken. Example: 
Screenshot from 2023-11-23 18 23 18
The overlay doesn't show properly on the dialog. 
I'm trying to figure out the culprit.

File: src/pages/bridge/TransferDialog.tsx [hunk 1]
```diff
@@ -69,66 +76,72 @@ function BaseTransferDialog(props: BaseTransferDialogProps) {
...
                   variant="short"
-                  publicKey={vaultStellarPublicKey}
+                  publicKey={destinationStellarAddress}
                 />
               </div>
-            )}
-            {showMemo && (
               <div className="flex flex-row justify-between">
-                <div className="text-sm">Memo</div>
+                <div className="text-sm">Vault PEN Address ({tenantNameCapitalized})</div>
```
Comment (ebma) [inline src/pages/bridge/TransferDialog.tsx:109]: We should not call it PEN address. I would actually suggest we completely remove that and only call it `Vault Address ({tenant})`.

File: src/pages/bridge/TransferDialog.tsx [hunk 2]
```diff
@@ -69,66 +76,72 @@ function BaseTransferDialog(props: BaseTransferDialogProps) {
...
                   className="text-sm p-0"
                   variant="short"
-                  publicKey={expectedStellarMemo}
+                  publicKey={transfer.original.vault.accountId.toString()}
                 />
               </div>
-            )}
+              {vaultStellarPublicKey && (
+                <div className="flex flex-row justify-between">
+                  <div className="text-sm">Vault PEN Address</div>
```
Comment (ebma) [inline src/pages/bridge/TransferDialog.tsx:119]: This is the vault's Stellar address and not a PEN address.

**Review by ebma — CHANGES_REQUESTED** [review]
I know that it's phrased similarly in the designs but we should change the namings.

**Review by ebma — APPROVED** [review]
Looks good. But I noticed that something is wrong with the pop up for the active account in the top-right hand corner. It doesn't show properly anymore, not sure if this was introduced in this PR or another

To reproduce, connect to a wallet account and then click on your account in the top right corner. It will show a pop-up but it seems like it is placed above the button and not below it.

**Pull request merged**